### DISCUSSION TITLE
Is it possible to personalize the right side panel in connect modal (image attached) ?

### DISCUSSION THREAD

**Discussion by ajaynarang** [rainbow-me/rainbowkit#1401] [Q&A] [answered]
Is it possible to personalize the section below? I would like to incorporate my own content and icons into it.

<img width="654" alt="image" src="https://github.com/rainbow-me/rainbowkit/assets/8322989/bca9afd3-abfc-4056-b59b-d7716b5751b4">

**Comment by DanielSinclair** [comment, marked as answer]
This is not currently possible. But you can reduce the visibility of this UI with compact mode described [here](https://www.rainbowkit.com/docs/modal-sizes).
